### PR TITLE
T365195 docker image docs 2

### PR DESCRIPTION
**Pull request by rti**
depends on #695

### CODE REVIEW

File: build/QuickStatements/Dockerfile [hunk 1]
```diff
@@ -43,9 +43,13 @@ ENV APACHE_DOCUMENT_ROOT /var/www/html/quickstatements/public_html
 RUN sed -ri -e "s!/var/www/html!${APACHE_DOCUMENT_ROOT}!g" /etc/apache2/sites-available/*.conf && \
     sed -ri -e "s!/var/www/!${APACHE_DOCUMENT_ROOT}!g" /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
 
-ENV MW_WG_SITENAME=${MW_WG_SITENAME}\
-    MW_WG_LANGUAGE_CODE=${MW_WG_LANGUAGE_CODE}\
-    PHP_TIMEZONE=UTC
+ENV SITENAME=wikibase \
+    MW_WG_LANGUAGE_CODE=en \
```
Comment (lorenjohnson) [inline build/QuickStatements/Dockerfile:47]: Rename...

File: build.sh [hunk 2]
```diff
@@ -77,8 +77,8 @@ function build_wikibase {
         --build-arg WIKIBASEEDTF_COMMIT="$WIKIBASEEDTF_COMMIT" \
         --build-arg WIKIBASELOCALMEDIA_COMMIT="$WIKIBASELOCALMEDIA_COMMIT" \
         \
-        --build-arg MW_WG_SITENAME="$MW_WG_SITENAME" \
-        --build-arg MW_WG_LANGUAGE_CODE="$MW_WG_LANGUAGE_CODE" \
+        --build-arg MW_WG_SITENAME="wikibase" \
```
Comment (lorenjohnson) [inline build.sh:80]: Maybe just remove this as it defaults in Dockerfile

File: build/QuickStatements/Dockerfile (hunk 1 above)
Comment (rti) [inline build/QuickStatements/Dockerfile:47]:
```suggestion
   LANGUAGE_CODE=en \
```

File: build.sh [hunk 3]
```diff
@@ -77,8 +77,8 @@ function build_wikibase {
...
+        --build-arg MW_WG_LANGUAGE_CODE="en" \
```
Comment (lorenjohnson) [inline build.sh:81]: Maybe just remove this as it defaults in Dockerfile

Comment (rti) [inline build.sh:81]: remove build args, check defaults in dockerfile

File: build.sh [hunk 4]
```diff
@@ -176,8 +176,8 @@ function build_quickstatements {
         --build-arg QUICKSTATEMENTS_COMMIT="$QUICKSTATEMENTS_COMMIT" \
         --build-arg MAGNUSTOOLS_COMMIT="$MAGNUSTOOLS_COMMIT" \
         \
-        --build-arg MW_WG_SITENAME="$MW_WG_SITENAME" \
-        --build-arg MW_WG_LANGUAGE_CODE="$MW_WG_LANGUAGE_CODE" \
+        --build-arg SITENAME="wikibase" \
+        --build-arg LANGUAGE_CODE="en" \
```
Comment (rti) [inline build.sh:180]: remove build args, check defaults in dockerfile

File: build.sh (hunk 3 above)
Comment (lorenjohnson) [reply build.sh:81]: Same for all of the below similar entries. Maybe we come back to this in a PR that keeps all of the default like this just in the Dockefiles.

**Review by lorenjohnson — APPROVED** [review]
LGTM! We will be looping back on these docs more in later releases, but this continues to be a great improvement from where we started.

File: build/WDQS/README.md [hunk 5]
```diff
@@ -21,42 +187,38 @@ If you can't use [RecentChanges], you'll need to reload from an RDF dump:
...
-| `/wdqs/allowlist.txt` | SPARQL endpoints allowed for federation |
-| `/wdqs/RWStore.properties` | Properties for the service |
+| File                         | Description                                                                                    |
+| ---------------------------- | ---------------------------------------------------------------------------------------------- |
+| `/wdqs/allowlist.txt`        | SPARQL endpoints allowed for federation                                                        |
+| `/wdqs/RWStore.properties`   | Properties for the service                                                                     |
 | `/templates/mwservices.json` | Template for MediaWiki services (populated and placed into `/wdqs/mwservices.json` at runtime) |
 
-### Troubleshooting
+## Known Issues
```
Comment (lorenjohnson) [inline build/WDQS/README.md:200]: Great for getting this in there!

File: build/QuickStatements/Dockerfile (hunk 1 above)
Comment (rti) [reply build/QuickStatements/Dockerfile:47]: https://phabricator.wikimedia.org/T368145

File: build.sh (hunk 3 above)
Comment (rti) [reply build.sh:81]: https://phabricator.wikimedia.org/T368145

Comment (rti) [reply build.sh:81]: https://phabricator.wikimedia.org/T368145

File: build.sh (hunk 4 above)
Comment (rti) [reply build.sh:180]: https://phabricator.wikimedia.org/T368145

**Comment by rti** [comment]
@datn decided to merge the branch to main already, please create a new PR for your changes from there.

**Pull request merged**